dist/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "EOS",
   "version": "1.0.0",
   "description": "EOS plugins.",
-  "summary": "EOS plugins.",
+  "summary": "Thank you for using EOS Icons. For any issues, feel free to check out https://eos-icons.com/.",
   "languages": ["en"],
   "author": "Abhirup Pal",
   "helpUrl": "abhiruppalmethodist@gmail.com",
```

Comment:
please change the `helpUrl` to the repo URL 
https://github.com/EOS-uiux-Solutions/eos-icons-adobe-xd